docs/user-guide.rst

```diff
@@ -147,6 +147,29 @@ You can specify headers as a dictionary in the ``headers`` argument in :meth:`~p
     >>> json.loads(r.data.decode('utf-8'))['headers']
     {'X-Something': 'value', ...}
 
+Or you can use the ``HTTPHeaderDict`` class to create a proper variable for storing header information.
+
+.. code-block:: pycon
+
+    >>> from urllib3 import HTTPHeaderDict
+    >>> Headers = HTTPHeaderDict()
+    >>> Headers.add('X-Something', 'value')
+    >>> Headers.add('Y-Something', 'value')
```

Comment:
Let's use a multi-value HTTP header here, for example Accept with both application/json and text/plain